### DISCUSSION TITLE
flet 0.20.0 mpv error

### DISCUSSION THREAD

**Discussion by burhansvural** [flet-dev/flet#2639] [Q&A] [answered]
### Question

error while loading shared libraries: libmpv.so.1: cannot open shared object file: No such file or directory

I'm using mpv 0.35.1-4
os: linux
kernel: 6.1.0-17-amd64

### Error message

```bash
flet: error while loading shared libraries: libmpv.so.1: cannot open shared object file: No such file or directory
```

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by 7576457** [comment]
I have the same problem, although I pre-download libmpv-dev

**Reply by burhansvural** [reply]
@Klu1d  I solved the problem. The problem is solved when you install the Mpv1 version, that is the old version.

**Reply by burhansvural** [reply, marked as answer]
@Klu1d use this file https://drive.google.com/file/d/1mxYxhcCfigZ55xI9xT0ntd9f5QNSyfCD/view?usp=sharing place this file in the /usr/local/lib/ directory and then use the sudo ln -s /usr/local/lib/libmpv.so.1 /usr/lib/libmpv.so.1 command

**Comment by FeodorFitsner** [comment]
What do you mean by "pre-download"?

Please try installing dependencies with:

```
sudo apt install libmpv-dev mpv
```

**Reply by burhansvural** [reply]
@FeodorFitsner I created the file that can fix this problem. Those using Debian operating system should place this file in the /usr/local/lib/ directory and then use the sudo ln -s /usr/local/lib/libmpv.so.1 /usr/lib/libmpv.so.1 command.

**Reply by FeodorFitsner** [reply]
I've been wondering if we could put that file inside Flet Linux distribution?

**Reply by burhansvural** [reply]
https://drive.google.com/file/d/1mxYxhcCfigZ55xI9xT0ntd9f5QNSyfCD/view?usp=sharing

**Reply by burhansvural** [reply]
Yes, it would actually be great to place the file in the flet linux distribution.

**Reply by GabyDs** [reply]
Excuse me, do you know where to find the libmujs.so package? After resolving for libmpv.so.1, this error emerged and I can't locate it for download. Thank you.

**Comment by bmcgonag** [comment]
I'm also experiencing this error, but on Fedora 39.  Any ideas what I should do.  I attempted the solution provided by @burhansvural , and have run

`sudo dnf install mpv-devel.x86_64 mpv`

I still get the error when I try `flet run` on any project.

Any help is greatly appreciated.

**Reply by nexusvoyager** [reply]
Tried the fix, same issue, help would be nice :bow:

**Reply by alejrot** [reply]
I had this trouble too in Fedora 39. I've solved it creating the symlink but in  **`/usr/lib64/`** instead of `/usr/lib/`:

`sudo ln -s /usr/lib64/libmpv.so /usr/lib64/libmpv.so.1`

This is assuming  `libmpv.so` exists, as is described above:

https://gist.github.com/luizmarinhojr/f5ede39febd53fceb757eef88618f2b8
